MergerLogic/ImageProcessing/TileScaler.cs

```diff
@@ -9,25 +9,52 @@ public class TileScaler : ITileScaler
         private const int TILE_WIDTH = 256;
         private const int TILE_HEIGHT = 256;
 
-        public void Upscale(MagickImage baseImage, Tile baseTile, Coord targetCoords)
+        public MagickImage Upscale(MagickImage baseImage, Tile baseTile, Coord targetCoords)
```

Comment:
add comment that includes description and link to chosen upscale algorithm

Reply:
the "algorithm" is just to enlarge the pixels...  its soo simple and basic calling it "algorithm" is overstatement and its trivial enough that I doubt the will be dedicated page anywhere (and linking to page with multiple algorithms is more confusing then helpful in this context